hack/run-local.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+# run-local.sh - run/cleanup the operator with OLM
+#
+# USAGE
+#    run-local.sh -a run/cleanup
+# OPTIONS
+#    -a=     Action     run/cleanup the operator installation
+#    -d      Debug      builds the operator image without using existing cache
+#?
+
+# container tool to use with operator-sdk
+CONTAINER_TOOL=podman
+
+function error-exit() {
+    echo "Error: $*" >&2
+    exit 1
+}
+
+# Options
+while getopts ":da:" opt; do
+    case "$opt" in
+	a) action="$OPTARG" ;;
+	d) debug="--image-build-args=\"--no-cache\"";;
+	?) error-exit "Unknown option"
+    esac
+done
+
+if [[ ! "$action" =~ ^run|cleanup$ ]]; then
+    error-exit "-a Action must be \"run\" or \"cleanup\""
+fi
+
+if [ -z "$AWS_SHARED_CREDENTIALS_FILE" ]; then
+    error-exit "env AWS_SHARED_CREDENTIALS_FILE not found"
+fi
+
+if [ -z "$KUBE_SSH_KEY_PATH" ]; then
+    error-exit "env KUBE_SSH_KEY_PATH not found"
+fi
+
+if [ -z "$CONTAINER_REPO" ]; then
+    error-exit "env CONTAINER_REPO not found"
+fi
+
+WMCO_ROOT=$(dirname "${BASH_SOURCE}")/..
+source $WMCO_ROOT/hack/common.sh
+
+cd $WMCO_ROOT
+OSDK=$(get_operator_sdk)
+
+# Builds the container image and pushes it to repository
+# Uses this built image to run the operator on the cluster
+# containers are tagged by branch name and it is user's responsibility to clean old/unused containers in
+# container repository as well as local system.
+case "$action" in
+    run)
+
+  $OSDK generate csv
+  if [ $? -ne 0 ] ; then
+      error-exit "failed to generate CSV for operator"
+  fi
```

Comment:
added it here so that, if someone changes the roles, it'll be reflected. But seems like a overkill and takes 1-2 seconds to execute as well. so i'll be removing it. and keeping it upto user to generate csv in case of any updates.